test/mysql.test.js

```diff
@@ -425,6 +425,45 @@ describe('mysql', function() {
     });
   });
 
+  it('should handle null in inq operator', function(done) {
+    Post.create({title: 'Foo', content: 'Bar'}, function(err, post) {
+      should.not.exist(err);
+      Post.find({where: {id: {inq: [null, 1]}}}, function(err, posts) {
```

Comment:
nitpick: maybe we can turn off `forceId`, then create the instance with `id`, to make sure the idValue we put in filter is the one we created.

Same for the following two use cases.